ui/tests/e2e/test.js

```diff
@@ -33,12 +33,33 @@ describe("End-to-end", () => {
     await assertFacet("Gender", "1018", "male", "518");
     await assertFacet("Total Exome Sequence", "707", "0B-10B", "435");
 
-    // Make sure second Super Population facet value is gray.
+    // Make sure non-selected facets are gray.
```

Comment:
Throughout this repro, "facet" is the whole facet box, "facet value" is one line inside the box ("Punjabi in Lahore,Pakistan").
```
// Make sure unselected facet values are gray.
```
Same below

Reply:
Done.